src/wpool_queue_manager.erl

```diff
@@ -311,9 +360,9 @@ handle_call(
       case erlang:is_process_alive(ClientPid) andalso
         Expires > now_in_microseconds() of
         true  ->
-          Reply = wpool_fsm_process:sync_send_event(Worker, Event),
-          gen_server:reply(Client, Reply),
-          {noreply, State#state{workers = NewWorkers}};
+          NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),
```

Comment:
Line 363 is too long:           NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),.